compiler_opt/es/blackbox_optimizers.py

```diff
@@ -64,6 +65,8 @@
 
 from compiler_opt.es import gradient_ascent_optimization_algorithms
 
+ArrayOfFloatArrays = Sequence[npt.NDArray[np.float32]]
```

Comment:
Nit - Might as well also have a `FloatArray = npt.NDArray[np.float32]]` and then `ArrayOfFloatArrays = Sequence[FloatArray]`  - just for consistency, right now there are various npt.NDArray[np.float32] around.